tests/index.spec.js

```diff
@@ -243,4 +243,26 @@ describe('Progress', () => {
     expect(circle.find(Circle).props().percent).toEqual([20, 20, 20, 20]);
     circle.unmount();
   });
+
+  it('should support indeterminate mode', () => {
+    const Indeterminate = () => {
+      return (
+        <>
+          <Circle />
+          <Line />
```

Comment:
How about `<Circle loading />` `<Line loading />`?

Reply:
Okay, so with the property **loading** we keep 40% as default  value when we dont define the percent property, what do you think ?

Reply:
Well i think the percent should be dynamic with 0 as default.

Reply:
> How about `<Circle loading />` `<Line loading />`?

@afc163 just added the loading property check it out 😉 , Thanks !

Reply:
> How about `<Circle loading />` `<Line loading />`?

@afc163 Could you please explain the reasons behind your proposal?

I strongly prefer the original behavior that the indeterminate mode is active if no value is specified. The DX is also better:

- An additional prop would make the interface more complicated. The mode and state of the progress would depend on two variables and no longer just one, making it more error-prone.

- The original behavior also fits much better into the use cases and to the APIs I can think of and use. If there is no value (yet), it is `undefined`. On the other hand, setting a value and still wanting to activate the indeterminate mode is a very rare use case. However, implementing this would be a single liner in the component using the progress if needed.

- Specifying the mode via the value prop would be equivalent to doing it on the native progress tag of html:
  > To make a determinate progress bar, add a [value](https://html.spec.whatwg.org/multipage/form-elements.html#attr-progress-value) attribute with the current progress [...]. To make an indeterminate progress bar, remove the [value](https://html.spec.whatwg.org/multipage/form-elements.html#attr-progress-value) attribute.
  
  https://html.spec.whatwg.org/multipage/form-elements.html#the-progress-element

Either way, `loading` is a too generic name in my opinion. A progress, which has a value, is also "loading". This would make it less intuitive and rather confuse when looking at it without knowing the docs.